template/web/app/mint/_components/ContractDemo.tsx

```diff
@@ -71,14 +77,19 @@ export default function MintContractDemo() {
   return (
     <div className="gap-16 lg:flex">
       <div className="w-full flex-shrink-0 flex-grow lg:max-w-[400px] xl:max-w-[600px]">
-        <NextImage
-          src={imageAddress}
-          altText={collectionName}
-          className="block w-full rounded-2xl"
-        />
+        {collectionImageUrl ? (
+          <NextImage
```

Comment:
yeah, more I read this, and more i keep thinking OnchainKit should have a NftMedia component.